pyartcd/pyartcd/jenkins.py

```diff
@@ -330,20 +330,23 @@ def start_rhcos(build_version: str, new_build: bool, **kwargs) -> Optional[str]:
 
 
 def start_build_sync(build_version: str, assembly: str, doozer_data_path: Optional[str] = None,
```

Comment:
Think we can add `dry_run` here?

Reply:
Not sure if it makes sense to auto-trigger dry run mode build-syncs. I'd rather skip the job in that case. Done with 64d543c8d4718bcf5a3aabcf14334ffa820249c5